runtimes/staking/src/pallets_consensus.rs

```diff
@@ -156,20 +156,20 @@ impl pallet_session::historical::Config for Runtime {
 
 // TODO::Have to fine tune parameters for practical use-case
 parameter_types! {
-    // 27 Days
-    pub const SlashDeferDuration: sp_staking::SessionIndex = 27 * 6;
-    pub const MinSelectedValidators: u32 = 5;
-    pub const MaxNominatorsPerValidator: u32 = 25;
-    pub const MaxValidatorPerNominator: u32 = 25;
-    pub const DefaultValidatorFee: Perbill = Perbill::from_percent(20);
+    // 6 Days
+    pub const SlashDeferDuration: sp_staking::SessionIndex = 6 * 6;
+    pub const MinSelectedValidators: u32 = 10;
+    pub const MaxNominatorsPerValidator: u32 = 100;
+    pub const MaxValidatorPerNominator: u32 = 10;
+    pub const DefaultValidatorFee: Perbill = Perbill::from_percent(5);
     pub const DefaultSlashRewardProportion: Perbill = Perbill::from_percent(10);
     pub const DefaultSlashRewardFraction: Perbill = Perbill::from_percent(50);
-    pub const DefaultStakingMaxValidators: u32 = 50;
-    pub const DefaultStakingMinStakeSessionSelection: Balance = 10 * constants::MILLICENTS;
-    pub const DefaultStakingMinValidatorBond: Balance = 10 * constants::MILLICENTS;
-    pub const DefaultStakingMinNominatorTotalBond: Balance = 10 * constants::MILLICENTS;
-    pub const DefaultStakingMinNominationChillThreshold: Balance = 3 * constants::MILLICENTS;
-    pub const MaxChunkUnlock: usize = 32;
+    pub const DefaultStakingMaxValidators: u32 = 20;
+    pub const DefaultStakingMinStakeSessionSelection: Balance = (3 * 10i32.pow(6)) as Balance * constants::NODL;
```

Comment:
I think this is clearer
```suggestion
    pub const DefaultStakingMinStakeSessionSelection: Balance = 3_000_000 as Balance * constants::NODL;
```